workalendar/europe/austria.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from workalendar.core import WesternCalendar, ChristianMixin
+
+
+class Austria(WesternCalendar, ChristianMixin):
+    "Austria"
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (5,   1, "National Holiday"),  # Staatsfeiertag
```

Comment:
right, we'll keep it that way until someone finds a better wording ; what's important is the date, the label can wait.